internal/testpb/impl.go

```diff
@@ -62,6 +62,8 @@ func (s *testServer) Multiple(stream Foo_MultipleServer) error {
 	}
 }
 
+func (*testServer) mustEmbedUnimplementedFooServer() {}
```

Comment:
a comment explaining why this is needed would be useful.

Reply:
done